Backend/info-skjerm-api/README.md

```diff
@@ -0,0 +1,36 @@
+# Backend Guide
+
+The info-skjerm backend is responsible retrieval, processing and storage of data. 
+The program is mainly a rest api that has various endpoints (more info under ##endpoints).
+
+The default port is 5237
+
+
+## prerequisites 
+
+You must have .net 8 installed.
+
+## Launching the backend
+
+To launch the application: 
+
+1. Navigate to ..\info-skjerm\Backend\info-skjerm-api in your terminal
+2. Run the command: "dotnet run build"
+3. To open the swagger page type /swagger after the url.
+
+## shutting down the backend
+
+To shut down the application:
+
+1. Enter the terminal
+2. press: ctrl + c
+
+## For developers 
+
+To be able to use the events endpoints you need a appsettings.json file in your project with a connection string like this:
```

Comment:
Since this is required for the backend and frontend to work correctly, I would move this to a category called *Setup* rather than *For developers*. I can make this change myself